Paco/src/com/pacoapp/paco/js/bridge/JavascriptEventLoader.java

```diff
@@ -56,18 +56,95 @@ public String loadAllEvents() {
     return convertExperimentResultsToJsonString;
   }
 
+  //TODO: check if method has references
   @JavascriptInterface
   public String getLastEvent() {
-    // TODO make this class manage retrieval better so that we aren't pulling tons of data into the webview.
-    List<Event> events = experimentProviderUtil.loadEventsForExperimentByServerId(experiment.getId());
-    return FeedbackActivity.convertLastEventToJsonString(events);
+	  return getLastNEvents("1");
+  }
+  
+  @JavascriptInterface
+  public String getLastNEvents(String numberOfRecords) {
+	// TODO: Should this be 10; Adding a default value of 10
```

Comment:
Please use the coding standard of two-space indentation. I will send you our Eclipse formatting.xml file.